npcs/carolina_jones.md

```diff
@@ -0,0 +1,12 @@
+# Carolina Tomes
+
+Affiliations: Indepenent
+
+Status: Missing
+
+![Carolina_Jones](/img/npcs/Carolina_Jones.png)
+
+**Background:** Carolina Tomes, archeologist extraordinaire and occasional lecturer at the Institute of Magic, was researching  the relics and history related to the Holy Caramel Empire. On the mission right before her disappearance, 'Carolina' was in possession of a staff that once belonged to the Holy Caramel Emperor. We learned that the real Carolina had gone missing and have not heard or seen 'Carolina' since. 
```

Comment:
```suggestion
##Background

Carolina Tomes, archeologist extraordinaire and occasional lecturer at the Institute of Magic, was researching  the relics and history related to the Holy Caramel Empire. On the mission right before her disappearance, 'Carolina' was in possession of a staff that once belonged to the Holy Caramel Emperor. We learned that the real Carolina had gone missing and have not heard or seen 'Carolina' since. 
```

Utilitize Markdown formatting